// format-as-you-type/dist/_lib/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "findIndexAfterMovement", {
  enumerable: true,
  get: function get() {
    return _findIndexAfterMovement.default;
  }
});
Object.defineProperty(exports, "handleBackspace", {
  enumerable: true,
  get: function get() {
    return _handleBackspace.default;
  }
});
Object.defineProperty(exports, "isLetter", {
  enumerable: true,
  get: function get() {
    return _isLetter.default;
  }
});
Object.defineProperty(exports, "isNumber", {
  enumerable: true,
  get: function get() {
    return _isNumber.default;
  }
});
Object.defineProperty(exports, "isSeparator", {
  enumerable: true,
  get: function get() {
    return _isSeparator.default;
  }
});
Object.defineProperty(exports, "removeCharacters", {
  enumerable: true,
  get: function get() {
    return _removeCharacters.default;
  }
});
Object.defineProperty(exports, "setDefaultOption", {
  enumerable: true,
  get: function get() {
    return _setDefaultOption.default;
  }
});

var _findIndexAfterMovement = _interopRequireDefault(require("./findIndexAfterMovement"));

var _handleBackspace = _interopRequireDefault(require("./handleBackspace"));

var _isLetter = _interopRequireDefault(require("./isLetter"));

var _isNumber = _interopRequireDefault(require("./isNumber"));

var _isSeparator = _interopRequireDefault(require("./isSeparator"));

var _removeCharacters = _interopRequireDefault(require("./removeCharacters"));

var _setDefaultOption = _interopRequireDefault(require("./setDefaultOption"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
