// format-as-you-type/dist/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "default", {
  enumerable: true,
  get: function get() {
    return _useFieldFormatter.default;
  }
});
Object.defineProperty(exports, "formatBigNumber", {
  enumerable: true,
  get: function get() {
    return _formatBigNumber.default;
  }
});
Object.defineProperty(exports, "formatCreditCard", {
  enumerable: true,
  get: function get() {
    return _formatCreditCard.default;
  }
});
Object.defineProperty(exports, "formatDate", {
  enumerable: true,
  get: function get() {
    return _formatDate.default;
  }
});
Object.defineProperty(exports, "formatPhone", {
  enumerable: true,
  get: function get() {
    return _formatPhone.default;
  }
});
Object.defineProperty(exports, "formatPostalCodeCanada", {
  enumerable: true,
  get: function get() {
    return _formatPostalCodeCanada.default;
  }
});
Object.defineProperty(exports, "formatString", {
  enumerable: true,
  get: function get() {
    return _formatString.default;
  }
});

var _formatBigNumber = _interopRequireDefault(require("./formatBigNumber"));

var _formatCreditCard = _interopRequireDefault(require("./formatCreditCard"));

var _formatDate = _interopRequireDefault(require("./formatDate"));

var _formatPhone = _interopRequireDefault(require("./formatPhone"));

var _formatPostalCodeCanada = _interopRequireDefault(require("./formatPostalCodeCanada"));

var _formatString = _interopRequireDefault(require("./formatString"));

var _useFieldFormatter = _interopRequireDefault(require("./useFieldFormatter"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
